src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -14,9 +21,12 @@
 import seedu.address.logic.parser.exceptions.ParseException;
 import seedu.address.model.person.Address;
 import seedu.address.model.person.Email;
+import seedu.address.model.person.Gender;
+import seedu.address.model.person.Ic;
```

Comment:
small nit but should we make this NRIC to be consistent with the CliSyntax.java? anything tbh

Reply:
I agree with Tanveer